bitnami/etcd/templates/cronjob-defrag.yaml

```
@@ -0,0 +1,92 @@
+  {{- end }}
+  concurrencyPolicy: {{ .Values.defrag.cronjob.concurrencyPolicy }}
+  schedule: {{ .Values.defrag.cronjob.schedule | quote }}
+  suspend: {{ .Values.defrag.cronjob.suspend }}
+  successfulJobsHistoryLimit: {{ .Values.defrag.cronjob.successfulJobsHistoryLimit }}
+  failedJobsHistoryLimit: {{ .Values.defrag.cronjob.failedJobsHistoryLimit }}
+  jobTemplate:
+    spec:
+      template:
+        metadata:
+          {{- $mergedLabels := mergeOverwrite (dict) .Values.commonLabels .Values.defrag.cronjob.podLabels -}}
+          labels: {{- include "common.labels.standard" ( dict "customLabels" $mergedLabels "context" $ ) | nindent 10 }}
+          {{- if .Values.defrag.cronjob.podAnnotations }}
+          annotations: {{- include "common.tplvalues.render" ( dict "value" .Values.defrag.cronjob.podAnnotations "context" $) | nindent 12 }}
+          {{- end }}
+        spec:
+          {{- if .Values.defrag.cronjob.activeDeadlineSeconds }}
+          activeDeadlineSeconds: {{ .Values.defrag.cronjob.activeDeadlineSeconds }}
+          {{- end }}
+          restartPolicy: {{ .Values.defrag.cronjob.restartPolicy }}
+          {{- if .Values.podSecurityContext.enabled }}
+          securityContext: {{- include "common.compatibility.renderSecurityContext" (dict "secContext" .Values.podSecurityContext "context" $) | nindent 12 }}
+          {{- end }}
+          containers:
+            - name: etcd-defrag
+              image: {{ include "etcd.image" . }}
+              imagePullPolicy: "IfNotPresent"
+              {{- if .Values.containerSecurityContext.enabled }}
+              securityContext: {{- include "common.compatibility.renderSecurityContext" (dict "secContext" .Values.containerSecurityContext "context" $) | nindent 16 }}
+              {{- end }}
+              env:
+                - name: BITNAMI_DEBUG
+                  value: {{ ternary "true" "false" (or .Values.image.debug .Values.diagnosticMode.enabled) | quote }}
+                - name: ETCDCTL_API
+                  value: "3"
+                - name: ETCD_ON_K8S
+                  value: "yes"
+                - name: MY_STS_NAME
+                  value: {{ include "common.names.fullname" . | quote }}
+                {{- $releaseNamespace := .Release.Namespace }}
+                {{- $etcdFullname := include "common.names.fullname" . }}
+                {{- $etcdHeadlessServiceName := (printf "%s-%s" $etcdFullname "headless" | trunc 63 | trimSuffix "-") }}
+                {{- $clusterDomain := .Values.clusterDomain }}
+                - name: ETCD_CLUSTER_DOMAIN
+                  value: {{ printf "%s.%s.svc.%s" $etcdHeadlessServiceName $releaseNamespace $clusterDomain | quote }}
+              {{- if .Values.defrag.cronjob.command }}
+              command: {{ .Values.defrag.cronjob.command | toYaml | nindent 16 }}
+              {{- else }}
+              command:
+                - etcdctl
+                - defrag
+                - --cluster
```

Split this into `command` and `args`:
```suggestion
              {{- if .Values.defrag.cronjob.command }}
              command: {{ .Values.defrag.cronjob.command | toYaml | nindent 16 }}
              {{- else }}
              command: ["etcdctl"]
              {{- end }}
              {{- if .Values.defrag.cronjob.args }}
              args: {{ .Values.defrag.cronjob.args | toYaml | nindent 16 }}
              {{- else }}
              args:
                - defrag
                - --cluster
```